packages/core/state/metadata/logics.ts

```diff
@@ -20,14 +21,25 @@ const requestAnnotations = createLogic({
         const { getState, httpClient } = deps;
         const applicationVersion = interaction.selectors.getApplicationVersion(getState());
         const baseUrl = interaction.selectors.getFileExplorerServiceBaseUrl(getState());
+        const displayAnnotations = selection.selectors.getAnnotationsToDisplay(getState());
         const annotationService = new AnnotationService({
             applicationVersion,
             baseUrl,
             httpClient,
         });
 
+        let annotations: Annotation[] = [];
+
         try {
-            const annotations = await annotationService.fetchAnnotations();
+            annotations = await annotationService.fetchAnnotations();
+            dispatch(receiveAnnotations(annotations));
+        } catch (err) {
+            console.error("Failed to fetch annotations", err);
+            done();
+            return;
+        }
+
+        if (!displayAnnotations.length) {
```

Comment:
When I initially loaded this branch locally, got an error on load that caused the app to fail because `displayAnnotations` was set to `[undefined]` which technically has length 1. Brian and I were able to special case around it temporarily, but since then haven't been able to replicate the error. Wanted to check if anyone else (@SeanLeRoy ?) has seen this bug locally?